appserver/admin/production_domain_template_web/pom.xml

```diff
@@ -135,26 +135,9 @@
                 <artifactId>xml-maven-plugin</artifactId>
                 <executions>
                     <execution>
-                        <goals>
-                            <goal>validate</goal>
-                        </goals>
-                        <phase>validate</phase>
+                        <id>validate-xml-files</id>
                     </execution>
                 </executions>
-                <configuration>
```

Comment:
Replaced by default execution configuration - which validates all xml files in given validation root directory, with several exceptions (templates which are not valid xml files, because contain placeholders not passing the validation or are empty).